examples/typescript/index.ts

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+
+const app: import("express").Application = module.exports = express();
```

Comment:
Why are you using this statement? you can easily set app value like this `const app = express();`

Reply:
Speaking from memory, I believe it was due to the narrowing and the [strictness](https://www.typescriptlang.org/docs/handbook/2/basic-types.html#strictness) of using the `@types/express` `npm` package. Overall, it's to my understanding there's a few issues with [`@types/expess`](https://github.com/DefinitelyTyped/DefinitelyTyped).

Again speaking from memory, I thought I had read in the documentation that `express` is 100% `http` compliant (relating to the built-in node.js library); however, there's loss of the exposed `HTTP.Server` types. Example

```ts
import express from "express";

import type HTTP from "http";
import type { Express } from "express-serve-static-core";

const Application: Express & HTTP.Server = express();
```

Does not allow compilation.

---

Regardless, I certainly can simplify